bert-text-classifier/src/js/sentimentanalysis.ts

```diff
@@ -0,0 +1,51 @@
+import WordPieceTokenizer from "../tokenization";
+import * as tf from '@tensorflow/tfjs';
+
+const vocabUrl = 'http://s3.us.cloud-object-storage.appdomain.cloud/bert-sentiment-tfjs/model/vocab.json'
```

Comment:
can you upload the files to the MAX specific bucket in COS (i.e., `/max-assets-prod/max-text-sentiment-classifier/tfjs/0.1.0/`)? let me know if you need assistance getting it into the correct bucket/location.